fivox/uriHandler.h

```diff
@@ -65,6 +65,12 @@ class URIHandler
     const brion::GIDSet& getGIDs() const;
 
     /**
+     * @return the GIDs from the 'pretarget' parameter, used for synapse
+     *         projections. Empty if parameter not specified.
+     */
+    const brion::GIDSet& getPreGIDs() const;
```

Comment:
I guess the post GIDs can be retrieved with getGIDs, the documentation is missing.